### DISCUSSION TITLE
Requesting access to repository settings

### DISCUSSION THREAD

**Discussion by Salkimmich** [veracruz-project/veracruz#669] [Ideas]
Hello Veracruz Maintainers:

I would like to run [CLOMonitor](https://clomonitor.io/search?page=1) checks for each repository, but requesting direct access to repository settings so I can run these checks locally without exposing your project to a publicly available dashboard. 

I'm reaching out to request access to the repository settings for Veracruz. Many best practices have already been established within the project, but I am preparing for future updates and enhancements to further strengthen security, compliance, and collaboration efforts.

To effectively review and enhance the project's security, compliance, and collaboration efforts, I require ownership or administrative privileges within the repository. This level of access will enable me to conduct a thorough review, identify areas for improvement, and propose enhancements that align with the project's objectives. Once this review is complete, you are welcome to remove these privileges, and alternatively, I can help you to establish these by assisting your administrators with direct implementation. 

Granting me access to the repository settings will enable me to review the existing configurations, identify any areas for improvement, and implement additional best practices and other measures to enhance the development workflow and project governance. Best practices can be highly specific to each project's unique setup and requirements, so our approach will be flexible and tailored to accommodate the Veracruz's specific needs.

Following review, any proposed changes to the process will be discussed transparently with the project maintainers and contributors. The standard PR and Issue process will be used to ensure full visibility and collaboration on all recommendations.

I look forward to collaborating with you for future updates!

Thanks,
Sal
CCC Technical Community Architect

**Comment by dreemkiller** [comment]
Before we can give you that, we need some confirmation that you are who you say you are. Please attend one of the public Veracruz meetings on Thursdays at 3:00 UTC so we can discuss this.

**Reply by Salkimmich** [reply]
Absolutely, I'll join in on the next call, and can walk you through these requests. Happy to preference least privelage. 
Can you confirm a meeting link, or add me to the calendar invite with this email: skimmich@contractor.linuxfoundation.org 

For reference, I've just joined the Confidential Computing Consortium as thier TCA: 
https://confidentialcomputing.io/about/leadership/
